src/components/Users.jsx

```diff
@@ -1,7 +1,46 @@
-import React from "react";
+import React, { useContext, useEffect, useState } from "react";
+import { UserContext } from "../contexts/user";
+import { getUsers } from "../utils/api";
+import "./Users.css"
 
 export const Users = () => {
+
+    const [users, setUsers] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+    const { setLoggedInUser } = useContext(UserContext)
+
+    useEffect(() => {
+        getUsers()
+            .then((users) => {
+                setUsers(users);
+                setIsLoading(false)
+            })
+    }, [])
+
+    if (isLoading) {
+        return <h1>Loading...</h1>
+    }
+
     return (
-        <p>This is users</p>
+        <div className="users">
+            <div className="col"></div>
+            <div className="content">
+                {users.map((user) => {
+                    return (
+                        <div className="user-list" key={user.name}>
+                            <div className="userCard">
+                                <h1>{user.username}</h1>
+                                <img className="profile-pic" src={user.avatar_url} />
+                                <br />
+                                <button onClick={() => setLoggedInUser(user)}>Log In !</button>
```

Comment:
WHen I click on a button to log in I get no feedback that the log in has been successful, please make sure we either redirect to a profile page / there is a message saying I'm logged in / anything else to that effect